### PR TITLE
Update plugin activation message styling

### LINKED ISSUE #3906: Plugin Activation Message Refinements, Part 2
## Feature Description

1. According to the design, the dark themed plugin activation message has a scrim at the bottom for all images.
1. "Welcome to the Web Stories for WordPress beta." should wrap better to the second line.

## Additional Context

Screenshot:

Screenshot 2020-08-14 at 11 39 32

CSS copied from Figma:

```css
background: linear-gradient(180deg, #020219 37.25%, rgba(28, 6, 25, 0) 100%);
transform: matrix(1, 0, 0, -1, 0, 0);
```

So something like that should work:

```css
&::after {
	content: "";
    display: block;
    background: linear-gradient(180deg, #020219 37.25%, rgba(28, 6, 25, 0) 100%);
    transform: matrix(1, 0, 0, -1, 0, 0);
    position: absolute;
    bottom: 0;
    left: 0;
    right: 0;
    width: 100%;
    height: 80px;
}
```

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Implementation Brief

* Add scrim
* Add `max-width: 270px;` on first container to improve text wrapping
* Add `max-width: 440px;` on the last container to improve text wrapping and spacing between 2nd and 3rd container

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

Updates the plugin activation notices as defined in [Figma](https://www.figma.com/file/EHkkPjNMRANg7RD6dKJ3QZ/Stable-Marketing?node-id=558%3A855), see https://github.com/google/web-stories-wp/issues/3906#issuecomment-689356994

## Relevant Technical Choices

* Added `srcset` attribute to images
* Adapted Figma design to make it responsive.  
  ⚠️ This is important, as the Figma design is > 1880px wide, but this notice needs to work on smaller screens too. It's quite possible and expected that the resulting layout will not look exactly as on Figma all the time.

## User-facing changes

Updated message upon plugin activation.

Some screenshots:
Screenshot 2020-09-21 at 11 18 17
Screenshot 2020-09-21 at 11 17 05
Screenshot 2020-09-21 at 11 16 58

Screenshot 2020-09-21 at 11 18 42

## Testing Instructions

1. Check out this branch
1. Run `npm run storybook`
1. Select "Plugin Activation" in sidebar
1. Run `npm run build`
1. Deactivate plugin
1. Activate plugin

_or_

1. Select this PR on the QA site
1. Deactivate plugin
1. Activate plugin

---

Fixes #3906

### CODE REVIEW

File: assets/src/activation-notice/app/components/image.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+`;
+
+function Image({ name, name2x, ...props }) {
+  const { cdnURL, isRTL } = useConfig();
+  return (
+    <Img
+      src={`${cdnURL}images/plugin-activation/${name}`}
+      srcSet={`${cdnURL}images/plugin-activation/${name}, ${cdnURL}images/plugin-activation/${name2x} 2x`}
+      alt=""
+      $rotationAngle={isRTL ? '-30deg' : '30deg'}
```
Comment (spacedmonkey) [inline assets/src/activation-notice/app/components/image.js:40]: ?

Comment (swissspidy) [reply assets/src/activation-notice/app/components/image.js:40]: What‘s unclear @spacedmonkey?

Props prefixed with a dollar sign are so-called transient props, a styled-components feature.

Comment (swissspidy) [reply assets/src/activation-notice/app/components/image.js:40]: See https://styled-components.com/docs/api#transient-props

Comment (spacedmonkey) [reply assets/src/activation-notice/app/components/image.js:40]: 👍 When a while since I looked at styled components. The more you know.

**Comment by spacedmonkey** [comment]
Screenshot 2020-09-21 at 18 50 21
I am seeing in my broswer. This is the URL I am seeing. Am I missing something? @swissspidy

**Comment by swissspidy** [comment]
@spacedmonkey were you able to test it now with the fixed URLs?

**Comment by spacedmonkey** [comment]
> @spacedmonkey were you able to test it now with the fixed URLs?

@swissspidy It still doesn't work locally. But if this is a known issue, I will approve this.

**Comment by swissspidy** [comment]
@spacedmonkey That seems isolated to your local testing. It works fine here and on the QA site.

**Review by spacedmonkey — APPROVED** [review]
Approved after chat with Pascal.

**Pull request merged**